### DISCUSSION TITLE
Add cursor-not-allowed to disabled buttons by default

### DISCUSSION THREAD

**Discussion by TreyWW** [saadeghi/daisyui#2554] [Ideas / feature requests]
I've came across this issue quite a few times, and my general solution has been to just wrap the button with cursor not allowed.

```html
❌This doesn't work
<button class="cursor-not-allowed btn btn-primary btn-disabled">button</button>

✔This does work
<div class="cursor-not-allowed">
    <button class="btn btn-primary btn-disabled">button</button>
</div>
```

But it would be nice to have btn-disabled automatically have the cursor blocked. An example is below.

No "disallowed" Cursor:
![no disallowed cursor](https://github.com/saadeghi/daisyui/assets/73353716/8616bef3-384a-4f6d-b8b2-65f8c8407fba)

Blocked Cursor:
![blocked cursor](https://github.com/saadeghi/daisyui/assets/73353716/7ef9e2ce-f787-43aa-bec1-94e5fc9804bd)

**Comment by saadeghi** [comment]
Unfortunately I couldn't find a way for this (except wrapping it with a div like you suggested)  
The problem is, we need `pointer-events: none` to disable the element but `pointer-events: none` disables everything including hover styles so `cursor: not-allowed` doesn't work.

Currently the only way is wrapping the button with a div like you suggested.

**Reply by TreyWW** [reply]
Ah i see! Thanks for the response, yeah that makes sense. I suppose wrapping it in a DIV is fine, thanks for letting me know though :)
